Modules/EventHub/2.0/deploy.json

```diff
@@ -1,140 +1,324 @@
 {
```

Comment:
@senthilkungumaraj did you guys come to an agreement with this module?

Reply:
Done we sync'd